changelog/1.20.0.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "1.20.0"
+description: "Released on 06/16/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.20.0.
+
+### Features ✨
+
+- web: Added the ability to set a site-wide workspace template policy at
+  **Manage > Admin > Templates > Template Policy**. If not set, Coder uses the
+  provided default.
+- web: Added the _nodeSelectors_ field to workspace templates.
+- other: Added a new JSON schema for writing Coder workspace as code templates
+  with code completion and syntax checking.
+- web: Added a service message that's displayed to all users of the system. The
+  message can be dismissed by each user at any point and will not be shown again
+  until there is a new message.
+- web: Added text wrapping to system banners.
+- infra: TLS management (/_TODO_/)
+- web: Added opt-in for HTTP Strict Transport Security. This setting can be
```

Comment:
Is this a feature or should it be listed under security?

Reply:
Looks like we have it listed twice; I'll remove from features since I think it should go with security (unless you think it should be the opposite?)

Reply:
It's a security thing, not a feature imo